src/components/profile/Recovery/ActiveMethod.js

```diff
@@ -155,7 +162,14 @@ class ActiveMethod extends Component {
                     {!deleteAllowed &&
                         <div className='not-allowed'>
                             <Translate id='recoveryMgmt.disableNotAllowed'/>
-                            <FormButton onClick={this.handleToggleDisable} type='button' className='small gray-blue'>
+                            <FormButton 
+                                onClick={() => {
+                                    Mixpanel.track(data.kind === 'phrase'? 'SR-SP Click close button': `SR Click close button for ${data.kind}`)
```

Comment:
Can we use `${data.kind}` all the time vs only for non seed phrase?

Reply:
since I use `SR-SP` for seed phrase and `SR` for non seed phrase, so I think it is good to keep it still